house_of_refuge/frontend/src/components/ResourceRow.js

```diff
@@ -235,29 +271,33 @@ export const ResourceRow = ({resource, isExpanded, onMatch, user, activeSub, com
         </tr>
         <tr>
           <th>Notatka</th>
-          {compact ? <td><EditableField value={note} onRename={updateNote}/></td> : <>
-            <td><EditableField value={note} onRename={updateNote}/></td>
-            <td className={"text-center"}>{getReadyDisplay(resource)}</td>
-            <td className={"text-center"} colSpan="3">
-              <Button size={"sm"}
-                      disabled={resource.status === "calling" && resource.owner.id !== user.id}
-                      onClick={() => {
-                        updateResource(
-                            resource, {status: "calling", owner: user.id},
-                            () => setShowModal(true));
-                      }}>
-                {resource.status === "calling" ? `DZWONI ${resource.owner?.display}` : "DZWONIĘ"}
-              </Button>
-            </td>
-          </>}
+          <td><EditableField value={note} onRename={updateNote}/></td>
+
+          <th>Status</th>
+          <td>{showStatus()}</td>
+
+          {compact ?
+              <s/>
+              :
+              <>
+                <td className={"text-center"}>{getReadyDisplay(resource)}</td>
+                <td className={"text-center"}>
+                  <Button size={"sm"}
+                          disabled={isTaken() && resource.owner.id !== user.id}
+                          onClick={openHost}>
+                    {isTaken() ? `DZWONI ${resource.owner?.display}` : "DZWONIĘ"}
```

Comment:
IIUC correctly, `isTaken` will return `true` also when current user is calling. So it seems to me `"DZWONIĘ"` will never get displayed. Can you clarify if I'm missing something?

Reply:
Ah, OK now I get it. the word "dzwonię" is a bit ambiguous, since it can mean "I am currently calling" as describing a fact about the present and, casually, an expression meaning "I want to call now". 

I guess if it's a clickable button it's clear enough :) Looking at the code it's not that evident. I would consider `"Chcę zadzwonić"` or `"Oznacz, że dzwonię"` but that may be too verbose.